### DISCUSSION TITLE
Tapo H200 Smart Hub

### DISCUSSION THREAD

**Discussion by pektommy** [petretiandrea/home-assistant-tapo-p100#580] [General]
Anyone managed to connect the H200 hub? I can't add the device, it won't find it when I try to add it by device ip. 
Error message "Failed to connect"

**Comment by hiltonhowie** [comment]
I have the same issue, checking the logs its trying to connect on port 80. I tried adding the ip address plus 192.168.60.20:443 but it made no difference

**Reply by zee36** [reply]
Hi All, same issue, can't connect to the H200 hub.

**Reply by Ossiez** [reply]
I'm also having the same issue, Any guidance on how to add it would be greatly appreciated.
Also, will it show any sensors that are connected to the hub as entities?

**Reply by raymen1** [reply]
Same here, trying to connect but have not been able to.

**Reply by ajock** [reply]
Hi, Is there any progress on the integration, waiting to connect h200 with switches and motion sensors to the home assistant

**Comment by petretiandrea** [comment]
H200 require a new protocol implementation I'm working on it

**Reply by f** [reply]
Hey @petretiandrea, is there a progress on it? Do you have any date to release H200 new protocol support? My setup desperately needs this implementation :)

**Reply by metafysikz** [reply]
I'm also patiently awaiting this feature :) For the meantime I'm using the H200 via SmartThings.

**Reply by ekareklam** [reply]
Hello. Is there any progress?

**Reply by Thomas-TB** [reply]
Hello, please do you already know a release date for an update that would connect H200?

**Reply by Mocusolvan** [reply]
I would also greatly appreciate the support for Tapo H200 Hub. Thanks

**Comment by ekareklam** [comment]
Hello, no sir.

13 Haz 2024 Per, saat 09:25 tarihinde Thomas-TB ***@***.***>
şunu yazdı:

**Reply by Thomas-TB** [reply]
It is a pity

**Comment by ekareklam** [comment]
I will try.


13 Haz 2024 Per, saat 11:26 tarihinde Thomas-TB ***@***.***>
şunu yazdı:

**Reply by Thomas-TB** [reply]
that's good news, thank you

**Comment by ekareklam** [comment]
Hi sir,
This stays on the screen.



15 Haz 2024 Cmt, saat 12:13 tarihinde ajock ***@***.***> şunu
yazdı:

**Comment by FaTonyNZ** [comment]
HI, appreciate the work towards H200 integration - wondering how this is going please? I made a rookie mistake and bought up large on Tapo only to realise when HA didn't automatically discover the hub... that it's not supported :-(.

**Comment by zee36** [comment]
Hi Guys, while this is being developed, you may go an alternative route setting up the H100/H200 hubs for a Matter device (go to the Tapo app and enable it, HASS, Google Home or SmartThings will all pick it up). 
In case you have Skyconnect or other Matter sticks (Conbee can make it too e.g.) connected to HASS, then sub-devices will coming through the Matter add-on in HASS. Currently I have entities for the Tapo Thermometer, Tapo motion sensors and Tapo contact sensors through Matter. I can see the Smart Buttons as well - but those are not triggering any actions yet. The flood sensors are not coming though yet.  As Matter under HASS is continuously evolving, more and more entities will be available. This is all local - no cloud required.     Hope this helps.

**Reply by FaTonyNZ** [reply]
Thanks Zee36. That's awesome news - thank you! I want to add motion sensors and contact sensors - so looks like these are going to work. Can I use a Sonoff zigbee dongle for this? I haven't done any matter before - so just checking I don't need to order a Conbee. Or is it that the H200 is Zigbee based, which is what a compatible dongle is required, not Matter (which I understand supports WiFi and Zigbee). thanks heaps for your help!

**Reply by Thomas-TB** [reply]
Thanks to Zee36, that works!

**Reply by zee36** [reply]
Hi FaTonyNZ, you will need a Matter enabled stick, or a Matter hub connected to your HASS server, then Matter and Thread add ons installed to HASS. Maybe there's a workaround to flash it to the Sonoff stick as well. 
Not sure about if Tapo uses Thread though, or something similar , I'm also just presuming they are using something similar radio protocol to Zigbee/Thread (I'm sure there are experts around the community who can advise on this)
Anyway, once the experimental Matter option is enabled in the Tapo app, it should go through to HASS, Google home and SmartThings (guess Apple's HomeKit will pick it up as well).

**Reply by FaTonyNZ** [reply]
Thanks Zee36, I've ordered a Conbee and will give this a go. 
Cheers Thomas-TB for confirming it works too.

**Reply by cagripolga** [reply]
I can't connect via Matter, Google Home doesn't see it, Apple Home doesn't see it. Is there anyone who can connect their H200 device to HA other than Matter?

**Comment by JohnMurphy474** [comment]
I connected the H200 by switching on the Bind to Matter settings in the hub and pairing with the HA Matter Server. All 20 devices came up in HA. Complete control of motion and contact sensors and Thermohygrometer for automations. However, no entities for S200B buttons and no ability to use the Siren on the H200. Reckon we have to wait for more Tapo updates to get more controls. As a result I've left all my alarm automations on the Tapo app an the rest of the automations in HA.

**Reply by FaTonyNZ** [reply]
Whoop ! That worked. Thanks

**Reply by JohnMurphy474** [reply]
@FaTonyNZ Did you go with a Conbee or did you do as I did with the HA Matter Server and binding the H200?

**Reply by FaTonyNZ** [reply]
I have the Conbee now and it's configured as the Zigbee controller. But I just used the HA Matter Server as you suggested and enabled Matter (Beta) on the hub. Then added a Matter device to get the Hub and motion sensors to show in HA.

**Reply by JohnMurphy474** [reply]
Good to know it worked for you too. I couldn't find a practical way to get it linked in to HA, and remembered the Matter link that Tapo had added in a recent firmware update. After that it was easy to find the Matter add-on for HA and link all (most) of the devices in. I just wish Tapo had added in some control over the hub itself (Siren etc.).

**Reply by p-afraz** [reply]
Unfortunately seems Tapo does not have any plans (at least for now) to enable any features on H200. They see it only as a Matter bridge and not a normal device.

https://community.tp-link.com/en/smart-home/forum/topic/695730

**Comment by hubgitmm** [comment]
Hi,
I connected hub H200 to Home Assisant HA Matter Server.
I have only one connected device, a door ring D230S1, and I can't see this device.

I integrated the hub with Tapo: Cameras Control (through HACS) and, in Tapo: Cameras Control (through HACS), I can see the tapo D230S1.

Any idea of how to view the D230S1 doorbell via the HA Matter Server integration ?

Thanks in advance.

**Comment by Henk-de-Wit** [comment]
H200 still connected to HA.

Unfortunately, the H200 cannot yet be connected to HA. I tried it with MATTER, but then the connections to the sensors installed on the H200 are unreliable.

In order to still be able to use the door sensor (and the water sensor), I wrote a small routine in the TAPO app. Door sensor open, operate P115 plug, etc.

I installed the plug in HA. So door sensor operated, P115 does something. You can use that in HA.

Just for the time being.....

**Reply by p-afraz** [reply]
I have it installed via Matter in HA and for T110 contact sensors, it has been quite stable without problems for me.

**Comment by DorwinDT** [comment]
Hello, still have problems with adding H200... I enter tplink credentials and "Failed to connect" is displayed.
via Matter I don't see any attached devices (water leak sensors)
any progress?
thank you

**Reply by hubgitmm** [reply]
Hello,
I face the same issue.
It's a real shame that Tapo, TP-Link, isn't doing anything to integrate the D230S1 doorbells, via matter, into Home Assistant...
